site-src/geps/gep-1619.md

```diff
@@ -0,0 +1,89 @@
+# GEP-1619: Session Persistence and Session Affinity
+
+* Issue: [#1619](https://github.com/kubernetes-sigs/gateway-api/issues/1619)
+* Status: Provisional
+
+(See definitions in [GEP Status][/contributing/gep#status].)
+
+## TLDR
+
+This GEP proposes a definition of session persistence and session affinity as the foundation of future session persistence related improvements. This GEP also outlines the ways session persistence could be achieved by implementations.
+
+## Goals
+- Define session persistence and session affinity to establish a common language
+- Identify differences in session persistence functionality between implementations
+
+## Non-Goals
+- Create proposals for adding session persistence configuration to Gateway API object specification
+- Mandate a default or expected session persistence functionality for implementations
+- Require implementations to support session persistence or a specific form of it
```

Comment:
If there are values that are broadly implementable, we _may_ want to consider this "core" conformance which would conflict with this non-goal. (I don't actually know that this is broadly implementable, but it may be safer to leave this non-goal out).